jenkinsci/platformlabeler-plugin
# Update test data

## Update test data

- Update Ubuntu 22.04 test data
- Add Clear Linux 39390 test data
- Add Amazon Linux 2023, remove Amazon Linux 2022
- Update tumbleweed data to 20230304
- Fix a typo in a comment

Update test data and documentation to match current operating systems.  Amazon Linux 2022 appears that it will never be general availabililty.  Replaced by Amazon Linux 2023.

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/jenkinsci/platformlabeler-plugin/blob/master/CONTRIBUTING.md) doc
- [x] I have referenced the Jira issue related to my changes in one or more commit messages
- [x] I have added tests that verify my changes
- [x] No spotbugs warnings were introduced with my changes
- [x] I have interactively tested my changes

## Types of changes

- [x] Test data and documentation
